modules/oadp-features-plugins-known-issues.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+// oadp-features-plugins-known-issues
+// * backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc
+// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc
+
+:_content-type: CONCEPT
```

Comment:
More of a note/reminder: "_content-type:" is now "_mod-docs-content-type:"

See Content Type Attributes: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#content-type-attributes